pkg/app/piped/notifier/slack.go

```diff
@@ -225,41 +225,62 @@ func (s *slack) buildSlackMessage(event model.NotificationEvent, webURL string)
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Triggered a new deployment for %q", md.Deployment.ApplicationName)
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_PLANNED:
 		md := event.Metadata.(*model.NotificationEventDeploymentPlanned)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Deployment for %q was planned", md.Deployment.ApplicationName)
 		text = md.Summary
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_WAIT_APPROVAL:
 		md := event.Metadata.(*model.NotificationEventDeploymentWaitApproval)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Deployment for %q is waiting for an approval", md.Deployment.ApplicationName)
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_APPROVED:
 		md := event.Metadata.(*model.NotificationEventDeploymentApproved)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Deployment for %q was approved", md.Deployment.ApplicationName)
 		text = fmt.Sprintf("Approved by %s", md.Approver)
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_SUCCEEDED:
 		md := event.Metadata.(*model.NotificationEventDeploymentSucceeded)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Deployment for %q was completed successfully", md.Deployment.ApplicationName)
 		color = slackSuccessColor
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_FAILED:
 		md := event.Metadata.(*model.NotificationEventDeploymentFailed)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
 		title = fmt.Sprintf("Deployment for %q was failed", md.Deployment.ApplicationName)
 		text = md.Reason
 		color = slackErrorColor
 		generateDeploymentEventData(md.Deployment, getAccountsAsString(md.MentionedAccounts))
 
 	case model.NotificationEventType_EVENT_DEPLOYMENT_CANCELLED:
 		md := event.Metadata.(*model.NotificationEventDeploymentCancelled)
+		if len(s.config.MentionedAccounts) != 0 {
+			md.MentionedAccounts = append(md.MentionedAccounts, s.config.MentionedAccounts...)
+		}
```

Comment:
Since this feature is not only for deployment events, mentioned accounts should be added to other notifications below, such as `NotificationEventType_EVENT_PIPED_STARTED`.